src/styles.css

```diff
@@ -2,6 +2,7 @@ body {
   height: 100vh;
   display: flex;
   /* TODO: Adjust CSS so that form is centered on page */
+  justify-content: center;
   align-items: center;
```

Comment:
Great job adding the `justify-content` property to center the form on the page! However, it would be helpful to add a comment explaining the purpose of the property for future reference.